templates/repo/issue/view_content/pull.tmpl

```diff
@@ -398,7 +399,7 @@
 										'allowed': {{$prUnit.PullRequestsConfig.AllowSquash}},
 										'textDoMerge': {{$.locale.Tr "repo.pulls.squash_merge_pull_request"}},
 										'mergeTitleFieldText': defaultSquashMergeTitle,
-										'mergeMessageFieldText': {{.GetCommitMessages}} + defaultMergeMessage,
+										'mergeMessageFieldText': {{.GetCommitMessages}} + defaultSquashMergeMessage,
```

Comment:
Why this changed?

Reply:
In the old code, defaultMergeMessage was always `'Reviewed-on: ' + {{$.Issue.HTMLURL}} + '\n' + {{$approvers}};`. So the `merge` type and the `squash` type could share it.

But now, it should come from templates, and there can be different templates for different types.